server/views/assessments/pages/sufficient-information/request-information.njk

```diff
@@ -0,0 +1,41 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% from "../../../partials/showErrorSummary.njk" import showErrorSummary %}
+{% from "../../../components/formFields/form-page-textarea/macro.njk" import formPageTextarea %}
+
+{% extends "../../../partials/layout.njk" %}
```

Comment:
Can we use the `questions` macro here?

Reply:
No, because we're POSTing to a different endpoint here 👍

Reply:
I assumed there was a reason we couldn't